Documentation/compilation.rst

```diff
@@ -16,34 +16,63 @@ In order to run SeisSol, you need to first install:
 Inital Adjustments to .bashrc
 -----------------------------
 
-Add the following lines to your .bashrc (vi ~/.bashrc).
+Create a new text file in your $HOME directory
 
 .. code-block:: bash
 
-  source /opt/intel/compiler/VERSION/bin/compilervars.sh intel64
+  cd $HOME
+  touch seissol_env.sh
+
+Open the file with your favorite text editor (vi ./seissol_env.sh) and add following lines:
+
+.. code-block:: bash
 
   # User specific aliases and functions
-  export PATH=$HOME/bin:$PATH
-  export LD_LIBRARY_PATH=$HOME/lib:$HOME/lib64:$LD_LIBRARY_PATH
+  export S3_HOME=$PWD/s3_software
+  
+  export PATH=$S3_HOME/bin:$PATH
+  export LD_LIBRARY_PATH=$S3_HOME/lib:$S3_HOME/lib64:$LD_LIBRARY_PATH  # lookup path for executables
+  export LIBRARY_PATH=$LD_LIBRARY_PATH:$LIBRARY_PATH  # lookup path for compilers
   export EDITOR=vi
-  export SCONS_LIB_DIR=$HOME/lib64/scons-2.2.0/
-  export PATH=$HOME/../libxsmm/bin:$PATH
-  export C_INCLUDE_PATH=$HOME/include:$C_INCLUDE_PATH 
-  export CPLUS_INCLUDE_PATH=$HOME/include:$CPLUS_INCLUDE_PATH
+  export SCONS_LIB_DIR=$S3_HOME/lib64/scons-2.2.0/
+  export C_INCLUDE_PATH=$S3_HOME/include:$C_INCLUDE_PATH
+  export CPLUS_INCLUDE_PATH=$S3_HOME/include:$CPLUS_INCLUDE_PATH
+  export PKG_CONFIG_PATH=$S3_HOME/lib/pkgconfig
+
+  export FC=mpiifort  # or mpifort
+  export CXX=mpiicpc  # or mpicxx
+  export CC=mpiicc    # or mpicc
```

Comment:
Probably not a good idea to set this in the bashrc file as it overrides user defined settings
Could be source of strange errors, better to set in each call separately